Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -304,14 +304,14 @@ void AnalyzeClassFeature(string key, PropertyDefinition property)
             {
                 if (_classAnalysisSetters.TryGetValue(key, out var featureFunc))
                 {
-                    var (shouldDelete, keyToSet) = featureFunc(property);
+                    var actionDriver = featureFunc(property);
```

Comment:
This var name is bugging me a little - can we rename to `analysisResult` or something simlar?